cirq/contrib/two_qubit_gates/example.py

```diff
@@ -0,0 +1,61 @@
+from time import time
```

Comment:
This file could use a top-level comment describing what it does and how to run it.

Reply:
Done